packages/app-store/feishucalendar/lib/CalendarService.ts

```diff
@@ -1,9 +1,10 @@
-import { getLocation, getRichDescription } from "@calcom/lib/CalEventParser";
```

Comment:
`getRichDescription` is moved to `CalendarManager` instead of being called in individual calendar apps.